src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.8.0-preview" targetFramework="net452" />
```

Comment:
Are you part of the resource manager client?  If so, we should just consider pushing the client into common code, as you don't want to be coupled to a particular version fo resource manager

Reply:
See previous comment - we're changing things so that we're not part of resource manager. Will update this PR to reflect that as soon as I can.